volume/struct.go

```diff
@@ -112,12 +119,142 @@ func NewVolumeEntry(req VolCreateRequest) *Volinfo {
 	v.StripeCount = req.StripeCount
 	v.DisperseCount = req.DisperseCount
 	v.RedundancyCount = req.RedundancyCount
+	//TODO : Generate internal username & password
 
-	v.Bricks = req.Bricks
-
+	v.Bricks = NewBrickEntries(req.Bricks, req.Force)
+	if v.Bricks == nil {
+		return nil
+	}
 	return v
 }
 
```

Comment:
You could move all the store functions into a new file in the same package. This keeps code nice and segmented.